content/en/blog/_posts/TBD-admission-controllers-for-container-drift/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Using Admission Controllers to Detect Container Drift at Runtime"
+date: TBD
+slug: admission-controllers-for-container-drift
+---
+
+**Author:** Saifuding Diliyaer (Box)
+{{< figure src="intro-illustration.png" alt="Intro Illustration" width="75%" attr="Illustration by Munire Aireti" >}}
```

Comment:
1. alt attribute fix
2. Don't set styles. If the figure's not showing at the right size though we can separately PR a CSS fix. We haven't so far added a mechanism to size figures that aren't also diagrams.

```suggestion
{{< figure src="intro-illustration.png" alt="Introductory illustration" attr="Illustration by Munire Aireti" >}}
```

Reply:
Updated as suggested.